pom.xml

```diff
@@ -170,6 +170,21 @@
             <argLine>@{argLine} -javaagent:${org.mockito:mockito-core:jar}</argLine>
           </configuration>
         </plugin>
+        <plugin>
+        <groupId>org.sonatype.central</groupId>
+        <artifactId>central-publishing-maven-plugin</artifactId>
+        <version>0.9.0</version>
+        <extensions>true</extensions>
+        <configuration>
+          <publishingServerId>central</publishingServerId>
+          <!-- Optional:
+          <autoPublish>false</autoPublish>
+          <waitUntil>validated</waitUntil>
+          -->
+        </configuration>
+      </plugin>
```

Comment:
This block should be indented once more to match the rest of the plugins. Also, we don't have this plugin in fcrepo itself, is it needed there too?